### PR TITLE
Minor naming changes in Shared FileSystems 4.20 Feature

### PR DESCRIPTION
**Pull request by abh1sar**
### Description

This PR fixes..
1. Change the UI tab name from Shared FIlesystem to Shared FileSystems
2. Rename Shared FileSystem VM to Shared FileSystem Instance
3. Remove create view table from schema-41910to42000 as it is not required.

Companion doc PR : https://github.com/apache/cloudstack-documentation/pull/435

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI
- [ ] test (unit or integration test code)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by abh1sar** [comment]
@blueorangutan package

**Comment by abh1sar** [comment]
@blueorangutan test

File: engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql [hunk 1]
```diff
@@ -307,204 +307,6 @@ CREATE TABLE `cloud`.`shared_filesystem`(
     INDEX `i_shared_filesystem__domain_id`(`domain_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
-DROP VIEW IF EXISTS `cloud`.`user_vm_view`;
```
Comment (sureshanaparti) [inline engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql:310]: @abh1sar this view updated for shared filesystem? not needed in upgrade path?

**Review by sureshanaparti — APPROVED** [review]
clgtm

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan package

File: engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql (hunk 1 above)
Comment (abh1sar) [reply engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql:310]: @sureshanaparti yes, the view is created in shared_filesystem_view.sql
We don't put any views in the upgrade path anymore.

View for user_vm_view was also udpdated in user_vm_view.sql. It was not needed here.

**Comment by abh1sar** [comment]
@JoaoJandre @rohityadavcloud 
This PR contains some minor cosmetic changes related to naming conventions for the Shared FileSystems feature we are shipping in 4.20. But I think it has high impact in terms of user experience and clarity.
For example,
- it is little jarring to see 'Shared FileSystem' as the only tab which is singular in UI. All other tabs ('Volumes', 'Snapshots' etc) are plural. 
- Cloudstack has the convention of calling user VM Instances, which we should maintain for Shared Filesystems VM as well.

The other small change is to just remove the user_vm_view definition from upgrade path, as we have separate files for creating views.

Can we please include this in 4.20.0?

**Comment by rohityadavcloud** [comment]
@abh1sar as long as this is reviewed and tested, I think it's okay but since freeze is declared - it's upto our release manager @JoaoJandre to review and approve/merge.

I like that we've dropped the unnecessary views changes as they're not kept in separate file, LGTM overall.

**Comment by weizhouapache** [comment]
@blueorangutan test

**Review by weizhouapache — COMMENTED** [review]
code lgtm

verified OK, no issues found

**Review by weizhouapache — APPROVED** [review]

**Comment by borisstoyanov** [comment]
@blueorangutan ui

**Review by borisstoyanov — APPROVED** [review]
LGTM

**Review by JoaoJandre — APPROVED** [review]
CLGTM. As this PR is just renaming some things and removing the duplicated view definition, I think we can include it.

**Pull request merged**